### ISSUE TITLE
WolfSSL Compatibility

### ISSUE THREAD

**Issue by castillofrancodamian** [getdnsapi/stubby#311]
Are there plans to make it compatible?

**Comment by saradickinson** [comment]
I don't believe we have any plans for this sorry, although we'ed be happy to look at a PR adding this.

The `getdns` library does contain a TLS abstraction layer to which enables support for both `openssl` and `gnutls` but there would have to be an analysis to see if all the functions needed are available (even `gnutls` doesn't fully support all the functionality in `libgetdns`).

@wtoorop (and Phillip) I think we talked about this a long time ago when considering embedded systems...

**Comment by birdie-github** [comment]
This issue is extremely relevant and important. I cannot press it but I just wanted to leave a comment and a confirmation.
